website/content/v1.3/kubernetes-guides/configuration/storage.md

```diff
@@ -107,6 +107,100 @@ talosctl -n <node ip> service kubelet restart
 
 Continue setting up [Mayastor](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor) using the official documentation.
 
+### Piraeus / LINSTOR
+
+* [Piraeus-Operator](https://piraeus.io/)
+* [LINSTOR](https://linbit.com/drbd/)
+* [DRBD Extension](https://github.com/siderolabs/extensions#storage)
+
+#### Install Piraeus Operator V2 (recommended, when starting from scratch!)
+
+There is already a how-to for Talos: [Link](https://github.com/piraeusdatastore/piraeus-operator/blob/v2/docs/how-to/talos.md)
+
+#### Install Piraeus Operator V1
+
+<!-- markdownlint-disable MD007 -->
+<!-- markdownlint-disable MD032 -->
+<!-- markdownlint-disable MD025 -->
+
+{{< tabpane lang="yaml" >}}
+{{< tab header="Strategic merge patch" >}}
+machine:
+  install:
+    extensions:
+      - image: ghcr.io/siderolabs/drbd:9.2.0-v1.3.0
+  kernel:
+    modules:
+      - name: drbd
+      - name: drbd_transport_tcp
+{{< /tab >}}
+{{< tab header="JSON patch" >}}
+- op: add
+  path: /machine/install/extensions/-
+  value:
+    - image: ghcr.io/siderolabs/drbd:9.2.0-v1.3.0
+- op: add
+  path: /machine/kernel/modules/-
+  value:
+    - name: drbd
+    - name: drbd_transport_tcp
+{{< /tab >}}
+{{< /tabpane >}}
+
+```sh
+# Apply config update on node
+talosctl patch mc --patch-file install-drbd.yml
+
+# Start upgrade of Talos with with current installed version to enable the extension
+talosctl upgrade --image=ghcr.io/siderolabs/installer:v1.3.0
```

Comment:
you can use `{{< release >}}` and it'll template the correct version

Reply:
How does the concrete command looke like?
I tried it by myself, bit it didn't work... I also did not find any hints in the documentation.

Reply:
I've updated the PR to use `{{< release >}}`